docs/the-new-architecture/landing-page.md

```diff
@@ -1,32 +1,223 @@
+
+### Learn more
+
+To achieve this, the New Architecture had to refactor multiple parts of the React Native infrastructure. To learn more about the refactor and other benefits it brings, check out the [documentation](https://github.com/reactwg/react-native-new-architecture) in the New Architecture working group.
+
+## What can I expect from enabling the New Architecture?
+
+While the New Architecture enables these features and improvements, enabling the New Architecture for your app or library may not immediately improve the performance or user experience.
+
+Your code has to be refactored to leverage new capabilities like synchronous layout effects and concurrent features. Although JSI will minimize the overhead between JavaScript and native memory, data serialization may not have been a bottleneck for your app's performance.
+
+Enabling the New Architecture in your app or library is opting into the future of React Native.
+
+The team is actively researching and developing new capabilities the New Architecture unlocks. For example, web alignment is an active area of exploration at Meta to be shipped to the React Native open source ecosystem.
+
+- [Updates to the event loop model](https://github.com/react-native-community/discussions-and-proposals/blob/main/proposals/0744-well-defined-event-loop.md)
+- [Node and layout APIs](https://github.com/react-native-community/discussions-and-proposals/blob/main/proposals/0607-dom-traversal-and-layout-apis.md)
+- [Styling and layout conformance](https://github.com/facebook/yoga/releases/tag/v2.0.0)
+
+You can follow along and contribute in our dedicated [discussions & proposals](https://github.com/react-native-community/discussions-and-proposals/discussions/651) repository.
+
+## Should I use the New Architecture today?
+
+Today, the New Architecture is considered experimental and we continue to refine backwards compatibility for a better adoption experience.
+
+The team plans to enable the New Architecture by default in an upcoming React Native release by the end of 2024.
+
+Our guidance is as follows
+
+- For most production apps, we do _not_ recommend enabling the New Architecture at this time. Waiting for the official will offer the best experience.
+- If you maintain a React Native library, we recommend enabling it and verifying your use-cases are covered. You can find the [instructions here](https://github.com/reactwg/react-native-new-architecture#guides).
+
+If you are interested in dogfooding the New Architecture experience, you can find [instructions](https://github.com/reactwg/react-native-new-architecture/blob/main/docs/enable-apps.md) in our dedicated working group. The [New Architecture working group](https://github.com/reactwg/react-native-new-architecture) is a dedicated space for support and coordination for New Architecture adoption and where the team posts regular updates.
```

Comment:
The only nit I have here:

I would create a:
```
### Enabling the New Architecture
```

heading before this paragraph as I can imagine is one of the thing folks will search for. I'm wondering if we even need the one-liner on how to do it in Android/iOS here or if we just want to let the user jump to the WG directly

Reply:
Let's keep the instructions in the WG for now, so we don't have to update multiple places